framework/test/unit/modules/token/cc_commands/cc_transfer.spec.ts

```diff
@@ -723,7 +738,7 @@ describe('CrossChain Transfer Command', () => {
 			});
 		});
 
-		it('should throw when the fee to initialize an account is insufficient', async () => {
+		it('should throw when escrow account has insufficient balance', async () => {
```

Comment:
I am a bit confused here. I don't see how balance of escrow connects with execute here.

Do you mean that the call of `initializeUserAccountInternal` fails (due to insufficient fee..)?

Reply:
I don't see any check for this case.